packages/react/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@logto/react",
+  "version": "0.1.0",
+  "source": "src/index.ts",
+  "main": "lib/index.js",
+  "exports": "./lib/index.js",
+  "typings": "lib/index.d.ts",
+  "license": "MPL-2.0",
+  "scripts": {
+    "preinstall": "npx only-allow pnpm",
+    "precommit": "lint-staged",
+    "build": "tsc",
+    "lint": "eslint --ext .ts --ext .tsx src",
+    "package": "parcel build",
+    "test": "jest",
+    "test:coverage": "jest --silent --coverage",
+    "test:report": "codecov -F react",
+    "prepublish": "pnpm build && pnpm package",
+    "report": "WITH_REPORT=true pnpm package"
+  },
+  "dependencies": {
+    "@logto/browser": "^0.1.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-router-dom": "^6.2.1"
+  },
+  "devDependencies": {
+    "@jest/types": "^27.5.1",
+    "@parcel/core": "^2.3.2",
+    "@silverhand/eslint-config": "^0.9.1",
+    "@silverhand/eslint-config-react": "^0.9.1",
+    "@silverhand/ts-config": "^0.9.1",
+    "@silverhand/ts-config-react": "^0.9.1",
+    "@testing-library/react": "^12.1.3",
+    "@testing-library/react-hooks": "^7.0.2",
+    "@types/jest": "^27.4.1",
+    "@types/react": "^17.0.39",
+    "@types/react-dom": "^17.0.11",
+    "@types/react-router-dom": "^5.3.3",
+    "codecov": "^3.8.3",
+    "eslint": "^8.9.0",
+    "jest": "^27.0.6",
+    "lint-staged": "^12.3.4",
+    "parcel": "^2.3.2",
+    "prettier": "^2.5.1",
+    "stylelint": "^13.13.1",
```

Comment:
I don' think we need to lint styles in react SDK. Consider removing it

Reply:
Removed.

(Used to fix missing peer in eslint-config-react. Seems to be unnecessary to fix it here.) 

![image](https://user-images.githubusercontent.com/10594507/155720726-7f2af063-7103-475a-864a-f99e410159ab.png)